scripts/devdeps.py

```diff
@@ -58,6 +59,9 @@ def depend_check(deps_name, *args):
         'websocket': 'websocket-client',
         'pytest_selenium': 'pytest-selenium',
         'pytest_cov': 'pytest-cov',
+        # 'mkl': 'mkl',
```

Comment:
mkl is definitely not a requirement, it is a completely optional add on for numpy